app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkAdapter.kt

```diff
@@ -186,6 +187,13 @@ class BookmarkAdapter(val emptyView: View, val interactor: BookmarkViewInteracto
                     true
                 } else false
             }
+
+            view.favicon.setOnClickListener({
```

Comment:
I'd prefer to use the `view.iconView` getter for this rather than directly using the favicon ID.